Library/ViewModels/PledgeViewModel.swift

```diff
@@ -3,47 +3,22 @@ import KsApi
 import Prelude
 import ReactiveSwift
 
-public typealias PledgeViewShippingRulesData = (
-  isEnabled: Bool,
-  isLoading: Bool,
-  selectedRule: ShippingRule?
-)
-
-public typealias PledgeViewData = (
-  project: Project,
-  reward: Reward,
-  isLoggedIn: Bool,
-  shipping: PledgeViewShippingRulesData,
-  pledgeTotal: Double
-)
+public typealias PledgeViewData = (project: Project, reward: Reward)
```

Comment:
Do we still need this `immortal` 😭 type? We're only passing `project` and `reward` so I would be OK with using `configureWithData` and passing it a tuple.

Don't feel strongly about this just wanted to ask?

Reply:
Similar to `configureSummaryViewControllerWithData` which uses `(Project, Double)`

Reply:
Sure let's be done with it once and for all 😂

Reply:
It was a nice type while we had it.